packages/hooks/src/useNewWalletCreation.ts

```diff
@@ -0,0 +1,41 @@
+import { useInvalidateQuery, useRequest } from '@deriv/api';
+import { useStore } from '@deriv/stores';
+import { useMemo } from 'react';
+
+/** A custom hook to create more wallets */
+const useNewWalletCreation = () => {
+    const { client, ui } = useStore();
+    const { accountRealReaction, switchAccount } = client;
+    const { toggleIsWalletCreationSuccessModalOpen } = ui;
+    const invalidate = useInvalidateQuery();
+    const { data, ...rest } = useRequest('new_account_wallet', {
+        onSuccess: async response => {
+            // Update the local storage account list with the new created wallet account.
+            await accountRealReaction(response);
```

Comment:
@thisyahlen-deriv Can we move this logic for `accountRealReaction` to a hook (or outside client-store)? I think it is not used anywhere 🤔.
Please correct me if I am wrong 🙏

Reply:
Its being used in client-store and plays a major part in account creation. I did this cause we already have existing logic to this.